sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -91,16 +90,16 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     constexpr static const char* c_HeaderLastModified = "last-modified";
     constexpr static const char* c_HeaderXMsNamespaceEnabled = "x-ms-namespace-enabled";
     constexpr static const char* c_HeaderXMsProperties = "x-ms-properties";
-    constexpr static const char* c_HeaderAcceptRanges = "accept-ranges";
-    constexpr static const char* c_HeaderContentRange = "content-range";
-    constexpr static const char* c_HeaderContentMD5 = "content-md5";
     constexpr static const char* c_HeaderPathLeaseAction = "x-ms-lease-action";
     constexpr static const char* c_HeaderXMsLeaseDuration = "x-ms-lease-duration";
     constexpr static const char* c_HeaderXMsLeaseBreakPeriod = "x-ms-lease-break-period";
     constexpr static const char* c_HeaderXMsLeaseId = "x-ms-lease-id";
     constexpr static const char* c_HeaderXMsLeaseTime = "x-ms-lease-time";
     constexpr static const char* c_HeaderRange = "range";
     constexpr static const char* c_HeaderXMsRangeGetContentMd5 = "x-ms-range-get-content-md5";
+    constexpr static const char* c_HeaderAcceptRanges = "accept-ranges";
```

Comment:
why did you reorder these?

Reply:
I didn't, the generator did that for some unknown reason. Don't have time to debug those at the moment, sorry.

Reply:
whatever